java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -85,10 +103,19 @@ public HttpConsumer(HttpConsumerConfig config, CountDownLatch messagesReceivedLa
     }
 
     public void createConsumer() throws IOException, InterruptedException, URISyntaxException {
-        String consumerInfo = "{\"name\":\"" + this.config.getClientId() + "\",\"format\":\"json\",\"auto.offset.reset\":\"earliest\"}";
+        Properties props = config.getProperties();
+        enableAutoCommit = config.getEnableAutoCommit();
+        Map<String, Object> map = new HashMap<>();
+        map.put("name", "my-consumer");
+        map.put("format", "json");
+        for (String k : commonProps) {
+            processProperty(props, k, map);
+        }
```

Comment:
Move the above 5 lines to createConsumerConfig() method

Reply:
Moved into `createConsumerConfig()`